plenum/test/view_change/test_view_change_by_current_state.py

```diff
@@ -0,0 +1,33 @@
+from plenum.test.pool_transactions.helper import disconnect_node_and_ensure_disconnected
+from plenum.test.test_node import ensureElectionsDone
+from plenum.test.view_change.helper import ensure_view_change, start_stopped_node
+
+nodeCount = 6
+
+
```

Comment:
I think the following integration test needs to be added:
Stop node; do view change to v1, do view change to v2 (in progress); start node; make sure that the restarted node is on v1 and not on v2.